src/transforms/remainder.jl

```diff
@@ -10,18 +10,78 @@ and returns a new table with an additional column containing the
 remainder value `xₙ₊₁ = total .- (x₁ + x₂ + ⋯ + xₙ)` If the `total`
 value is not specified, then default to the maximum sum across rows.
 """
-struct Remainder <: Transform end
+struct Remainder <: Transform
+  total::Union{Float64,Nothing}
+
+  function Remainder(total=nothing)
+    new(total)
+  end
```

Comment:
Please remove this inner constructor and add an outer constructor with `Remainder() = Remainder(nothing)`